web/src/pages/Transaction/index.tsx

```diff
@@ -1,19 +1,467 @@
+import { PanelDiv, BriefInfoDiv, InputOutPutTable, WithRowDiv } from './styled'
+
+import BlockHeightIcon from '../../asserts/block_height_green.png'
+import TimestampIcon from '../../asserts/timestamp_green.png'
+import TransactionIcon from '../../asserts/transaction_green.png'
+import VersionIcon from '../../asserts/version.png'
+import CopyGreenIcon from '../../asserts/copy_green.png'
+import CopyIcon from '../../asserts/copy.png'
+
+import { transaction, cell } from './mock'
+
+const cellData = cell.data
+const operationItems = ['Lock Script', 'Type Script', 'Data']
+
+const RowData = ({
+  type,
+  d,
+  isLast = false,
+  whichToLoad = null,
+  updateCellData,
+}: {
+  type: 'input' | 'output'
+  d: any
+  isLast?: boolean
+  whichToLoad?: 'Lock Script' | 'Type Script' | 'Data' | null
+  updateCellData: Function
+}) => {
+  const appContext = useContext(AppContext)
+
+  return (
+    <>
+      <tr className="tr-brief">
+        <td
+          style={{
+            width: 50,
+            fontSize: 18,
+            color: '#888888',
+          }}
+        >
+          {`#${d[`${type}_id`]}`}
+        </td>
+        <td
+          style={{
+            width: 1100 - 50 - 150 - 100 * 3,
+            fontSize: 16,
+            color: '#4bbc8e',
+          }}
+        >
+          {d.address_hash}
+        </td>
+        <td
+          style={{
+            width: 150,
+            fontSize: 16,
+            color: '#888888',
+          }}
+        >
+          {d.capacity}
+        </td>
+        {operationItems.map((item: string) => {
+          let className = 'td-operatable'
+          if (d.open === item) {
+            className += ' td-operatable-active '
+          }
+          return (
+            <td
+              key={item}
+              style={{
+                width: 100,
+                fontSize: 16,
+                color: '#4bbc8e',
+                fontWeight: 'bold',
+              }}
+            >
+              <div
+                role="button"
+                tabIndex={-1}
+                className={className}
+                onKeyPress={() => {}}
+                onClick={() => {
+                  const newD = {
+                    ...d,
+                  }
+                  if (newD.open === item) {
+                    newD.open = null
+                  } else {
+                    newD.open = item
+                    newD[item] = cellData
+                  }
+                  updateCellData(type, d[`${type}_id`], newD)
+                }}
+              >
+                {item}
+              </div>
+            </td>
+          )
+        })}
+      </tr>
+      {whichToLoad ? (
+        <tr
+          className="tr-detail"
+          style={{
+            borderBottom: !isLast ? '2px solid #4bbc8e' : 0,
+          }}
+        >
+          <td />
+          <td colSpan={5}>
+            <textarea
+              id={`textarea-${type}${+'-'}${d[`${type}_id`]}`}
+              style={{
```

Comment:
I think we should use more `styled-componnent` instead of style css as soon as possible.

Reply:
plan to try as much to not use style  directly in html-element, but sometimes to declare a variable with styled , which is a bit annoying

Reply:
I think we should have a consensus that use `styled component` or `style css`. Once we have a consensus, we should do it strictly. Otherwise code will be in confusion.